lorien/Misc/I18nParser.gd

```diff
@@ -51,12 +61,15 @@ func load_files() -> ParseResult:
 					
 					value = value.strip_edges()
 					value = templater.process_string(value)
+					value = value.replace("\\n", "\n")
```

Comment:
Made it possible to use \n escape codes in the translation files. It is a _bit_ hacky, but can be handy for layouting inside of default ConfirmDialogs without needing me (or anymone) to define a custom dialog just to fix the layouting.